doc/nrf/samples/benchmarks.rst

```diff
@@ -0,0 +1,11 @@
+.. _benchmark_samples:
+
+Benchmark samples
+#################
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Subpages
+   :glob:
+
+   ../../../samples/benchmarks/*/README
```

Comment:
Are there going to be more benchmark samples? If so, are they planned?

If the answer to one of these questions is no, please do not add a separate category, but include the sample under "Other samples". Categories should be introduced when we have 2+ components of a given type. Having categories for single components is just unnecessary overstructuring.

Reply:
FYI: agreed, we've already discussed this approach earlier and will be applying it. It seems there are no set in stone plans to incorporate any additional benchmark samples, at least for the next couple of months

Reply:
Like we have multicore/hello_world?

Reply:
For now I don't know if there are plans for more benchmarks to be moved here, but there is potential in it. It was decided to be put under samples/benchmarks/coremark in zephyr-next so I assume it is a correct place for it here.

Reply:
Also there is no Other samples category yet or am I mistaken?

Reply:
aa so you mean to put in only in doc other samples category and not to move whole code to other folder?

Reply:
If we move the documentation, we need to align the sample location to avoid user confusion. I am not sure if Other samples category is the right direction.

Should we change the current location for this sample in this code review? The sample folder location was somehow agreed upon a year ago during the review of this documentation part:

https://res.developer.nordicsemi.com/ncs/doc/latest/zephyr/samples/benchmarks/coremark/README.html

I think we need to take time before making any final decision on this one. CC: @pdunaj

Reply:
I take back the first sentence of my previous comment. It seems that documentation categories do not always correspond to the sample folder location. The other category is used to mix together samples from different folders:

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/other.html

Some examples:

https://github.com/nrfconnect/sdk-nrf/tree/main/samples/ipc
https://github.com/nrfconnect/sdk-nrf/tree/main/samples/app_event_manager
https://github.com/nrfconnect/sdk-nrf/tree/main/samples/bootloader
(...)